app/audit/ContractorNameAndEmployerRef.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit
+
+import play.api.libs.json.{Json, OWrites}
+
+case class ContractorNameAndEmployerRef(name: Option[String], ern: String)
+
+object ContractorNameAndEmployerRef {
+  implicit def writes: OWrites[ContractorNameAndEmployerRef] = Json.writes[ContractorNameAndEmployerRef]
```

Comment:
This method does not have a test, but I think it is tested indirectly.

Reply:
I'll add a test for it

Reply:
updated